app/gui/view/graph-editor/src/component/breadcrumbs.rs

```diff
@@ -206,7 +207,7 @@ impl BreadcrumbsModel {
         let background = background::View::new();
         let gap_width = default();
 
-        scene.layers.panel.add(&background);
+        scene.layers.panel_background.add(&background);
```

Comment:
Why do we need additional layer for it? Isn't shape order dependency enough to put it at the bottom?

Reply:
No, the layer is needed to put it into the correct order with respect to the `ExecutionModeSelector`and the `DropDownMenu`.